hazelcast-hibernate/hazelcast-hibernate3/src/main/java/com/hazelcast/hibernate/instance/HazelcastAccessor.java

```diff
@@ -69,10 +69,6 @@ public static HazelcastInstance getHazelcastInstance(final SessionFactory sessio
     public static HazelcastInstance getHazelcastInstance(final SessionFactoryImplementor sessionFactory) {
         final Settings settings = sessionFactory.getSettings();
         final RegionFactory rf = settings.getRegionFactory();
-        if (rf == null) {
-            LOGGER.severe("Hibernate 2nd level cache has not been enabled!");
-            return null;
-        }
```

Comment:
I think it is designed to be a utility class to access hazelcast instance used by hibernate. See the old google groups post about it : https://groups.google.com/forum/#!searchin/hazelcast/HazelcastAccessor%7Csort:relevance%7Cspell:false/hazelcast/kVZfayIhjYA/Voh66NHOo1IJ